packages/web-components/src/components/ic-empty-state/ic-empty-state.tsx

```diff
@@ -55,6 +55,11 @@ export class EmptyState {
    */
   @Prop() subheading?: string;
 
+  /**
+   * Sets the theme color to the dark or light theme color. "inherit" will set the color based on the system settings or ic-theme component.
+   */
+  @Prop() theme?: IcThemeMode = "inherit";
```

Comment:
Are we not using `light` as default setting?

Reply:
Just checked and all of the other components using the theme prop are defaulting to "inherit" e.g. tooltip, data list